fiorella/src/scripts/sections/menu.js

```diff
@@ -0,0 +1,10 @@
+var showMenu = document.getElementById("show-menu-btn");
+var hideMenu = document.getElementById("hide-menu-btn");
+
+showMenu.addEventListener('click', function() {
+  document.getElementById("menu").style.width = "100%";
```

Comment:
By convention... use single quotes instead of double quotes
And remember to use classes instead of modifying the inline style of the element

Reply:
Done! I already push another commit for a new ticket "Navigation bar" to this pull request, since it belongs to the header I thought it would be better to be in this branch, so I appreciate a Review on that commits. Thanks!